### PR TITLE
Create prestashop-81-translations.md

### PR DESCRIPTION
**Pull request by l-delin**
on standby until the 8.1 catalog is pushed

### CODE REVIEW

**Comment by l-delin** [comment]
The catalog has been pushed so this announcement is ready to be published 🥳  @coffrep @kpodemski

File: content/news/posts/2023/prestashop-81-translations.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+tags: [translation, i18n, l10n, "8.1"]
+---
+
+PrestaShop’s [8.1 beta release](https://build.prestashop-project.org/news/2023/prestashop-8-1-beta-release/) was announced a few months ago. The stable release is now just around the corner. While it brings new features, a new version also means new strings. A file containing all 8.1 strings has been pushed to Crowdin, and is ready for translation!
+
+## 8.1 strings are available for translation
+Translators, proofreaders… your project needs you! Feel free to take a look at those new strings and help your language reach 100% translation. 💪
+In this article, I will address the key elements of the translation process of this new catalog.
+
+### New strings & completion rates
```
Comment (coffrep) [inline content/news/posts/2023/prestashop-81-translations.md]:
```suggestion
### New strings and completion rates 
```

Comment (coffrep) [inline content/news/posts/2023/prestashop-81-translations.md]:
```suggestion
### New strings and completion rates 
```

File: content/news/posts/2023/prestashop-81-translations.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### Mistakes in the source strings
+Even though the wording is always reviewed when a pull request is submitted, some mistakes could have been missed. So, don’t hesitate to let us know if you notice something wrong so that we can correct it. Note that, despite the string freeze, we will wait for about 3 or 4 more weeks before adding the corrections to Crowdin. After that, we won’t be able to add any new wording. If you spot a mistake, please tag me (@lea.delin) in the “Comments” section of the string, or contact me directly!
+
+### Context
+The 8.1 version has not been released yet, so some strings may appear a bit unclear because of a lack of context. If this is the case, please don’t hesitate to ask for help! You can [open an issue](https://docs.prestashop-project.org/translating-prestashop/translating-prestashop-software-basics/translating-on-crowdin/the-dos-and-donts-of-crowdin-translation#use-the-comments-and-issue-features) in the “Comments” section of a string and I will do my best to answer your questions. In any case, don’t forget that you will still be able to edit the translations when the new version is released.
+
+### A note on variables & QA checker
```
Comment (coffrep) [inline content/news/posts/2023/prestashop-81-translations.md]:
```suggestion
### A note on variables and QA checker
```

File: content/news/posts/2023/prestashop-81-translations.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+If you don’t know what a variable is, it’s a value that can change depending on its context in the software. On Crowdin, these variables are highlighted in green in the source string and must **not be translated**, as they can result in major software bugs.
+In the Crowdin editor, you can simply click on a variable to copy and paste it into the target language. Always make sure to copy and paste the variable **exactly as displayed on the original string**!
+
+Note that the QA checker will be enabled in the project in the upcoming weeks. Variables will then be automatically checked in target translations to make sure no variables slip through the cracks anymore. If you wish to know more about these changes, keep an eye out for next Crowdin Monthly, or check out the PrestaShop Open Source project's Slack translation channel.
```
Comment (coffrep) [inline content/news/posts/2023/prestashop-81-translations.md]:
```suggestion
Note that the QA checker will be enabled in the project in the upcoming weeks. Variables will then be automatically checked in target translations to make sure no variables slip through the cracks anymore. If you wish to know more about these changes, keep an eye out for the next Crowdin Monthly, or check out the PrestaShop Open Source project's Slack translation channel.
```

**Review by coffrep — CHANGES_REQUESTED** [review]
Hi @l-delin, great post, I'm just adding a few suggestions.

**Comment by l-delin** [comment]
Thank you @coffrep ! I've added a section on QA checks since we enabled them on Crowdin right after the release of the catalog. Let me know what you think 😊

**Review by tleon — APPROVED** [review]

File: content/news/posts/2023/prestashop-81-translations.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+**”Incorrect translation” issues**: if an issue is open indicating that the “current translation is wrong”, a *warning message* will be displayed.
+
+**ICU syntax**: if the ICU message syntax isn’t correct, an *error message* will be displayed.
+
+**Duplicate translation**: if the translation already exists, an *error message* will be displayed.
+
+Note that since this is the first time QA checks are enabled for the project, some settings may still need adjustment. Of course, if you want to give your feedback or if you have any questions, feel free to contact me at translation@prestashop.com or on Crowdin ✉️
+
+If you wish to learn more about the QA checker and how it works, can check out the [Crowdin documentation](https://support.crowdin.com/qa-checks/).
```
Comment (coffrep) [inline content/news/posts/2023/prestashop-81-translations.md]:
```suggestion
If you wish to learn more about the QA checker and how it works, you can check out the [Crowdin documentation](https://support.crowdin.com/qa-checks/).
```

Comment (coffrep) [inline content/news/posts/2023/prestashop-81-translations.md]:
```suggestion
If you wish to learn more about the QA checker and how it works, you can check out the [Crowdin documentation](https://support.crowdin.com/qa-checks/).
```

**Review by coffrep — APPROVED** [review]
Hi @l-delin, I'm approving with one change.

**Comment by l-delin** [comment]
Done 👋

**Comment by kpodemski** [comment]
thanks @l-delin :)

**Pull request merged**